nnpackage/schema/circle_schema.fbs

```diff
@@ -35,6 +35,7 @@
 // Version 0.8: GRU op is added. UINT4 is added.
 // Version 0.9: GGML_Q{X}_{Y} types are added. Weight compression option is added.
 //              ROPE op is added.
+// Version 0.10: MXFP4, MXINT8 types are added.
```

Comment:
from previous update PR comment, https://github.com/Samsung/ONE/pull/14013#issuecomment-2370118318 , I think we have to stick to 0.9